packages/lodestar/test/unit/chain/stateTransition/util/misc.test.ts

```diff
@@ -28,17 +28,17 @@ describe("getDomain", () => {
 
   it("epoch before fork epoch should result in domain === previous fork version * 2**32 + domain type", () => {
     const result = getDomain(config, state, 4, 8);
-    assert.equal(result.toString('hex'), '0000000400000004');
+    assert.equal(result.toString('hex'), '0400000004000000');
   });
 
   it("epoch before fork epoch should result in domain === previous fork version * 2**32 + domain type", () => {
     const result = getDomain(config, state, 5, 13);
-    assert.equal(result.toString('hex'), '0000000500000005');
+    assert.equal(result.toString('hex'), '0500000005000000');
   });
 
   it("epoch before fork epoch should result in domain === previous fork version * 2**32 + domain type", () => {
     const result = getDomain(config, state, 5, 12);
-    assert.equal(result.toString('hex'), '0000000500000005');
+    assert.equal(result.toString('hex'), '0500000005000000');
```

Comment:
What changed that caused these to flip?

Reply:
See: https://github.com/ChainSafe/lodestar/commit/4b1143fad7d7ce17c9f50b3f6948afb07bc02024#diff-375d2bcfea48c91c150b334eca6354aaL19